### ISSUE TITLE
SpeedDial : Fix aria-controls values

### ISSUE THREAD

**Issue by kyybo** [primefaces/primereact#7110]
### Describe the bug

"The aria-controls attribute identifies the element (or elements) whose contents or presence are controlled by the element on which the attribute is set" (--> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls)

The aria-controls attribute should only be placed on the SpeedDial trigger button. And its value must be the identifier or list of identifiers of its items.

**Current behavior :**
<img width="757" alt="image" src="https://github.com/user-attachments/assets/8a5a1b7a-f23a-46c1-8902-95f39425bbb1">

### System Information

```Shell
"primereact": "10.8.2"
"react": "^18.2.0"
```

### Expected behavior

The aria-controls value should be an actual id (or a list of ids)

<img width="758" alt="image" src="https://github.com/user-attachments/assets/209adf11-a8b0-4171-ade8-ee140de55d98">

**Comment by kyybo** [comment]
PR : #7111
